pkgs/top-level/all-packages.nix

```diff
@@ -8563,7 +8563,24 @@ with pkgs;
 
   grpc-client-cli = callPackage ../development/tools/misc/grpc-client-cli { };
 
-  grub2 = callPackage ../tools/misc/grub/default.nix { };
+  grub2_upstream = callPackage ../tools/misc/grub/default.nix { };
+
+  # downgrade to 2.06 from 163b243c2a1c4ce666d69635a90e4e08cf27d899
+  grub2 = assert lib.asserts.assertMsg
```

Comment:
I would recommend calling this `grub2_06` and letting the call sites override like `grub2 = grub2_06`.

Reply:
I was worried about possibly breaking something by having inconsistent grub versions, so I thought this way would potentially be safer.

Reply:
Ok just a suggestion!